### ISSUE TITLE
Resizing browser window should resize backing texture.

### ISSUE THREAD

**Issue by bluemarvin** [MozillaReality/FirefoxReality#319]
Hardware: All
Steps to Reproduce:
1) Go into focus mode
2) Resize window from a corner
Actual Behavior: Size of browser content stays the same only the polygon gets larger.
Expected Behavior: Content resizes along with window.
Recommendation: Since we are already resizing the window when the aspect ratio changes, we need to resize the texture even when the aspect ratio stays constant. There needs to be a 1:1 mapping between the geometry size of the browser window and the texture rendered onto it.

**Comment by nancyhang** [comment]
This is linked to #274

**Comment by nancyhang** [comment]
Blocked by resize mode feature.

**Comment by kearwood** [comment]
Perhaps this behavior should vary depending on the "focus" or regular mode.  If resizing in normal mode, perhaps more pixels should be created.  In focus mode, perhaps the number of pixels per degree of FOV should be constant?

**Comment by kearwood** [comment]
If a video element or canvas element are "full screen", could we match the resolution natively?

**Comment by bluemarvin** [comment]
This still needs to be done. But we need to figure out what the correct behavior should be.

**Comment by nancyhang** [comment]
Blocked by #371

**Comment by larsbergstrom** [comment]
@bluemarvin This is a V1.1 feature, right?

**Comment by larsbergstrom** [comment]
@bluemarvin Is this still worth doing? I believe all the blocking resize code has landed.

**Comment by bluemarvin** [comment]
It's already done.
